### ISSUE TITLE
[Accessibility]: Contributors page: Button group "active" style is very similar to focus style

### ISSUE THREAD

**Issue by umpox** [sourcegraph/sourcegraph#35354]
### Audit type

General/Other

### User journey audit issue

https://github.com/sourcegraph/sourcegraph/issues/34427

### Problem description

<img width="546" alt="image" src="https://user-images.githubusercontent.com/9516420/168042167-7d828788-e278-40f2-bfb6-46fd871e1ce6.png">

Path is focused in this screenshot, "All time" is the active button.

### Expected behavior

It should be easier to distinguish between focus and active styles. cc: @sourcegraph/design

### Assigning labels

- [X] Please give this issue an estimate by applying a label like `estimate/Xd`, where X is the estimated number of days it will take to complete.
- [X] If this issue is specific to a specific Sourcegraph product, please assign the appropriate team label to this issue.
- [X] If this issue will require input from designers in order to complete, please assign the label `needs-design`.
- [ ] If you are confident that this issue should be fixed by GitStart, please assign the label `gitstart`.

### Owner

This issue will be fixed by my team, I have assigned a relevant member to it, or I will do so in the near future.

**Comment by umpox** [comment]
Also related issues: 
- https://github.com/sourcegraph/sourcegraph/issues/25218
- https://github.com/sourcegraph/sourcegraph/issues/23849

I have closed those in favour of this one.

We should ensure these buttons are updated everywhere to use the correct style. See comment from @AlicjaSuska: https://github.com/sourcegraph/sourcegraph/issues/23849#issuecomment-1084356829

**Comment by umpox** [comment]
@gitstart-sourcegraph Please can you take a look at this:

> Let's differentiate the active and focus states:
> - focus: as is
> - active: the same visually as hover state
